### PR TITLE
trip inference updates

### PR DESCRIPTION
**Pull request by unreasonableman**
- changes to allow running trip inference against version-controlled training data
- refactoring

### CODE REVIEW

File: inference.py [hunk 1]
```diff
@@ -9,15 +10,15 @@
 from segment import Segment
 from timer import Timer
 
-FEET_PER_MILE = 5280
 STOP_PROXIMITY = 150
-SCORE_THRESHOLD = 50
+SCORE_THRESHOLD = 7
```
Comment (owades) [inline inference.py:14]: I know almost nothing about score threshold but this seems low. Is the idea that a very low score is acceptable in some cases?

File: util.py [hunk 2]
```diff
@@ -127,6 +148,12 @@ def haversine_distance(lat1, lon1, lat2, lon2):
 
     return EARTH_RADIUS_IN_FEET * c
 
+def distance(x0, y0, x1, y1):
```
Comment (owades) [inline util.py:151]: get_distance_between_points() would be more descriptive

**Review by owades — APPROVED** [review]
See two small comments

File: inference.py (hunk 1 above)
Comment (unreasonableman) [reply inference.py:14]: The training data has allowed me to just play with different values for settings, and it turns out that a lower value here gives more accurate results overall.

File: util.py (hunk 2 above)
Comment (unreasonableman) [reply util.py:151]: Maybe, but this is just straight Pythagorean distance, and `distance()` or `length()` are commonly used names for this functionality.

**Pull request merged**